tests/Functional/User/RegistrationTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace App\Tests\Functional\User;
+
+use App\Factory\UserFactory;
+use App\Tests\FunctionalTestCase;
+
+class RegistrationTest extends FunctionalTestCase
+{
+    public function testRegistration(): void
+    {
+        $this->browser()
+            ->visit('/register')
+            ->fillField('Name', 'Karen Smith')
+            ->fillField('Email', 'ksmith@example.com')
+            ->fillField('Password', 'super s3cure password')
```

Comment:
WDYT about light silliness?

```suggestion
            ->fillField('Password', 'skynet84')
```